.github/workflows/test.yml

```
@@ -3,9 +3,14 @@ name: Unit tests
 on:
   # Run on PRs and pushes, only on significant changes.
   push:
+    branches:
+      - develop
+      - master
```

```suggestion
      - main
```

My colleagu changed the `master` to `main` branch in the meantime 🙈